tabbycat/motions/views.py

```diff
@@ -170,13 +171,31 @@ def post(self, request, *args, **kwargs):
 
 class ReleaseMotionsView(BaseReleaseMotionsView):
     edit_permission = Permission.RELEASE_MOTION
-
     action_log_type = ActionLogEntry.ActionType.MOTIONS_RELEASE
     motions_released = True
 
     @property
     def message_text(self):
         return ngettext("Released the motion.", "Released the motions.", self.round.motion_set.count())
+        
+        
+   
+
+    def post(self, request, *args, **kwargs):
+        tournament_preferences = self.tournament.preferences
+        preparation_time_preference = tournament_preferences.get('debate_rules__preparation_time', None)
+
+        start_minutes = preparation_time_preference  
+
+        
+        start_time = timezone.now() + timezone.timedelta(minutes=start_minutes)
+        start_time_local = start_time.astimezone(timezone.get_current_timezone())
+        self.round.starts_at = start_time_local
+        self.round.save()
+
+
+        messages.success(request, f"Debate will start at {start_time_local.strftime('%H:%M')} ({settings.TIME_ZONE})")
```

Comment:
```suggestion
        preparation_time = self.tournament.pref('preparation_time')
        
        if preparation_time > -1:
            self.round.starts_at = timezone.now() + timezone.timedelta(minutes=start_minutes)
            self.round.save()

            messages.success(request, f"Debates will start at {self.round.start_time.strftime('%H:%M')} ({settings.TIME_ZONE})")
```

Does this look good/work well? We have a `.pref()` shortcut, and as we're using `timezone.now()` it should already be in the proper TZ.

Reply:
I've fixed it. Please check that it's all OK now.